src/templates/index.ejs

```diff
@@ -17,7 +17,6 @@
     <meta name="author" content="<%= htmlWebpackPlugin.options.constants.author %>">
     <meta name="description" content="<%= htmlWebpackPlugin.options.constants.description %>">
     <meta property="og:title" content="<%= htmlWebpackPlugin.options.constants.title %>">
-    <link rel="icon" type="image/png" href="/images/favicon.png">
```

Comment:
Interesting. Ya, it appears to be working.

Reply:
Seems that has been fixed then, yay!

Good job updating to Webpack 5.x 🍾 — that unblocked CI options now with node16+ compatibility. (At least for #161 now that GH dropped support for older runners…) — I originally tried bumping stuff to latest 4.47.x that had crypto patched to support node16+ but this gets rid of way more legacy deps so definitely huge thanks for bringing it more up to date!

I have some loader updates to test and will propose them, and one major bump that will need node18+ eventually so that's TBD.

In the meantime I'll check compatibility with Node versions 16–18–20/LTS so we know if there are any limitations ahead of us if we want to drop EOL support and move on…

Once again, amazing job! Thanks 🎉